en/micro-integrator/docs/references/mediators/dB-Report-Mediator.md

```diff
@@ -5,6 +5,9 @@ The **DB Report Mediator** is similar to the [DBLookup Mediator](dBLookup-Mediat
 !!! Info
     The DB Report mediator is a [content-aware](../../../references/mediators/about-mediators/#classification-of-mediators) mediator.
 
+!!! Note
+    Currently, the 'DB-Report-Mediator' does not supporting the 'json-eval' expressions to extract the parameters.
```

Comment:
Suggestion: "Currently, the 'DB-Report-Mediator' does not support the 'json-eval' expressions to extract the parameters.